app/views/shared/_innovation_editor_footer.html.erb

```diff
@@ -0,0 +1,83 @@
+<%
+  if @practice
+    path_options = {
+      editors: {
+        back: nil,
+        continue: practice_introduction_path(@practice)
+      },
+      introduction: {
+        back: practice_editors_path(@practice),
+        continue: practice_adoptions_path(@practice)
+      },
+      adoptions: {
+        back: practice_introduction_path(@practice),
+        continue: practice_overview_path(@practice)
+      },
+      overview: {
+        back: practice_adoptions_path(@practice),
+        continue: practice_implementation_path(@practice)
+      },
+      implementation: {
+        back: practice_overview_path(@practice),
+        continue: practice_about_path(@practice)
+      },
+      about: {
+        back: practice_implementation_path(@practice),
+        continue: nil
+      }
+    }
+  else
+    path_options = {
+      description: {
+        back: nil,
+        continue: product_intrapreneur_path(@product)
+      },
+      intrapreneur: {
+        back: product_description_path(@product),
+        continue: product_multimedia_path(@product)
+      },
+      multimedia: {
+        back: product_intrapreneur_path(@product),
+        continue: nil
+      }
+    }
+  end
+%>
+
+<footer class="innovation-editor-footer <%= yield(:footer_classes) %>">
+  <%# only show return to top on mobile views, practice show page and long practice editor pages (per design 9/21/21) %>
+  <% if @show_return_to_top %>
+    <div class="grid-container usa-footer__return-to-top desktop:display-block">
+      <button class="dm-button--unstyled-primary dm-return-to-top width-auto">Return to Top</button>
+    </div>
+  <% else %>
+    <div class="grid-container usa-footer__return-to-top tablet:display-none">
+      <button class="dm-button--unstyled-primary dm-return-to-top width-auto">Return to Top</button>
+    </div>
+  <% end %>
```

Comment:
```suggestion
  <% if @show_return_to_top %>
    <div class="grid-container usa-footer__return-to-top desktop:display-block">
      <button class="dm-button--unstyled-primary dm-return-to-top width-auto">Return to Top</button>
    </div>
  <% end %>
```
This end of the conditional can be removed since we're inside of the innovation editor footer. The other cases (mobile views and practice show pages) are covered by `shared/_/footer.html.erb`.